packages/fluentui/react-bindings/src/styles/resolveStyles.ts

```diff
@@ -124,15 +125,13 @@ const resolveStyles = (
   // Our API should be aligned with it
```

Comment:
Should we remove/change this comment?

Reply:
Good catch, removed it and inlined `direction` to `rendererParam` object